server/apps/packages/package_service.py

```diff
@@ -146,6 +147,16 @@ def check_package_associations(self, docs):
             for assoc in associations:
                 self.extract_default_association_data(group, assoc)
 
+    def check_not_in_personal_space(self, docs):
+        for doc in docs:
```

Comment:
It would be good to add some documentation why we are adding this check for packages?